frontend/form-builder/src/components/ComponentSettings/ButtonGroup.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useContext } from "react";
+import classes from "./ComponentSettings.module.css";
+import { Button } from "@digdir/design-system-react";
+import { FloppydiskFillIcon, TrashFillIcon } from "@navikt/aksel-icons";
+import { FormBuilderContext } from "../../context";
+import { validateOrder } from "../../../../main-app/src/components/SubHeader/utils";
+import { useTranslation } from "react-i18next";
+
+export const ButtonGroup = () => {
+  const { t } = useTranslation();
+  const { form, setForm, currentComponent, setCurrentComponent } = useContext(FormBuilderContext);
+
+  const handleDeleteComponent = (index: number) => {
+    const newForm = form;
```

Comment:
Maybe we need here to make  a copy of form to not directly modify the `form `? 
`const newForm = { ...form };`